tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
I'll defer this question to @jzaefferer. I don't have an opinion where to set this in the tests. I can say I'm ok though using Globalize this way to generate for needed formatters. :+1: